### PR TITLE
unbreak Node.js 20

### PR DESCRIPTION
**Pull request by legobeat**
#487 (run ci on node v18 and v20) unearthed some regressions. This aims to resolve that.

- deps: bump `source-map` packages
  - [Failing test](https://github.com/LavaMoat/LavaMoat/pull/487#issuecomment-1536098619)
  - https://github.com/mozilla/source-map/issues/454

### CODE REVIEW

**Comment by legobeat** [comment]
Next error doesn't seem as easy as bumping deps

File: packages/node/package.json [hunk 1]
```diff
@@ -19,16 +19,16 @@
...
-    "json-stable-stringify": "^1.0.1",
+    "json-stable-stringify": "^1.0.2",
     "lavamoat-core": "^14.0.0",
     "lavamoat-tofu": "^6.0.2",
-    "node-gyp-build": "^4.2.3",
-    "object.fromentries": "^2.0.2",
-    "resolve": "^1.17.0",
-    "yargs": "^16.0.0"
+    "node-gyp-build": "^4.6.0",
+    "object.fromentries": "^2.0.6",
```
Comment (naugtur) [inline packages/node/package.json]: this should no longer be needed, right?

Comment (legobeat) [reply packages/node/package.json]: Indeed! Removed.

**Comment by legobeat** [comment]
Removed unrelated commits which have been folded into other currently open PRs.

**Comment by boneskull** [comment]
Since builds aren't kept forever, [here's a gist](https://gist.github.com/boneskull/81d6630db9c6af9ece1de068ff1922e7) of the failures.

**UPDATE** Maybe that's something else entirely. I'm not sure what's going on in that test.  It's from my rebased [copy of this branch](https://github.com/LavaMoat/LavaMoat/tree/boneskull/unbreak-node-18) which should have removed the engines restrictions...

**Comment by legobeat** [comment]
Superseded by #718

**Pull request closed without merging**